examples/persistent-volumes/volumes/local-01.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolume
+apiVersion: v1beta3
+metadata:
+  name: pv0001
+  labels:
+    type: local
+spec:
+  capacity:
+    storage: 10Gi
```

Comment:
It was originally sized.  "storage" was thought more specific.  I'm happy to change it to whatever the consensus is.